subsys/bluetooth/host/conn.c

```diff
@@ -1954,7 +1972,8 @@ struct bt_conn *bt_conn_create_le(const bt_addr_le_t *peer,
 {
 	struct bt_conn *conn;
 
-	if (!atomic_test_bit(bt_dev.flags, BT_DEV_READY)) {
+	if (!IS_ENABLED(CONFIG_BT_CONN_LOOPBACK) &&
```

Comment:
why? Shouldn't we respect the device having to be ready before we create a connection? this might have additional side-effects otherwise if parts of `hci_core` are not ready?

Reply:
We don't use the hci_core, in fact the test don't even have a driver.